src/__tests__/page-extensions/page-extensions.test.tsx

```diff
@@ -16,15 +16,16 @@ describe('page file extensions', () => {
     });
 
     describe('unknown extension', () => {
-      it('throws "page not found" error', async () => {
-        await expect(
-          getPage({
-            nextRoot,
-            route: '/invalid',
-          })
-        ).rejects.toThrow(
-          '[next-page-tester] No matching page found for given route'
-        );
+      it('throws "page not found" error page', async () => {
+        const { render } = await getPage({
+          nextRoot,
+          route: '/invalid',
+        });
+        render();
+        expect(screen.getByText('404')).toBeInTheDocument();
+        expect(
+          screen.getByText('This page could not be found', { exact: false })
+        ).toBeInTheDocument();
```

Comment:
Same as before: we might want to wrap up a test utility function to say "expect this to be not found page".